FFTGeneratorConfigs.scala

```diff
@@ -0,0 +1,23 @@
+package FFTGenerator
```

Comment:
Since these config fragments are the main way to parameterize the system, can you comment in detail in the code what each of these things do.